assets/src/components/AssignmentTable.js

```diff
@@ -322,190 +320,182 @@ function AssignmentTable (props) {
           </Tooltip>
         </Grid>
       </Grid>
-      <RootRef rootRef={tableRef}>
-        <TableContainer className={classes.container}>
-          <MTable stickyHeader ref={tableRef}>
-            <TableHead>
-              <TableRow>
-                {
-                  [
-                    'Week',
-                    'Due',
-                    'Type',
-                    'Name',
-                    'Percent of Final Grade',
-                    'Score / Out of',
-                    'Status',
-                    'Lock Goal'
-                  ].map((heading, key) => (
+      <TableContainer className={classes.container} ref={tableRef}>
+        <MTable stickyHeader>
+          <TableHead>
+            <TableRow>
```

Comment:
All the changes here are to replace `RootRef`, which is being deprecated in future releases of Material UI and is throwing a warning/error in the console. It's mostly unindenting things, adding the `ref` to `TableContainer`, and replacing the `ConditionalWrapper` logic with the ternary `ref` work on new line 352.